portfolio/src/main/webapp/martialArt.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>MMA</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/favicon.ico"/>
+    <link href="https://fonts.googleapis.com/css2?family=Nanum+Myeongjo:wght@700&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="header">
+        <a href="index.html" class="logo">Federick Gonzalez</a>
+        <div class="header-right">
+            <a href="index.html">Home</a>
+            <a class ="active" href="#">MMA</a>
+            <div class="dropdown">
+                <button class="dropbtn">More
+                    <i class="fa fa-caret-down"></i>
+                </button>
+                <div class="dropdown-content">
+                    <a href="photography.html">Photos</a>
+                     <a href="art.html">Art</a>
+                    <a href="profiles.html">Profiles</a>
+                </div> 
+            </div>
+        </div>
+    </div>
+    <div style="height:30px;"></div>
+
+    <div class="smallText">
+        <h1>Music and Martial Arts</h1>
+        <h3 style="text-align: left;">Music has been a big part of my life for as long as I can remember. Whether playing my trombone for school, going to orchestra and jazz practice on weekends, or listening to music while working (And even when writing this webpage), music has always been present. Currently, I am a member of the <a href="https://www.youtube.com/channel/UCAaUAFeRINTEPoMOsW3pdfg">Columbia Pops orchestra</a>, and I've included some of our amazing work below. More recently I have found a place as a member of <a href="https://www.youtube.com/user/ColumbiaWushu">Columbia Wushu</a>, a performance based martial arts group, though it is admittedly not my first experience in the realm of martial arts. These two groups make up the communities in which I've found most of my friends, and I'm glad to be a part of them! Below I have some pics and videos so you can have a better idea of what both groups entail!<h3>
```

Comment:
and here :)

Reply:
Text wrapping is proving to be detrimental to my readability, but it's been corrected!